src/fft.jl

```diff
@@ -44,7 +44,7 @@ function determine_fft_size(lattice::AbstractMatrix{T}, Ecut;
     fft_size
 end
 function determine_fft_size(model::Model, Ecut; kwargs...)
-    determine_fft_size(model.lattice, Ecut; kwargs...)
+    determine_fft_size(model.lattice, to_energy(Ecut); kwargs...)
```

Comment:
That's pretty low-level in the code. At that point I think we should assume that Ecut has already been converted to the proper value.